clicommand/kubernetes_bootstrap.go

```diff
@@ -164,6 +164,9 @@ var KubernetesBootstrapCommand = cli.Command{
 			if err != nil {
 				l.Error("Error waiting for client interrupt: %v", err)
 			}
+			// The context cancellation handler in process.Run first calls
+			// Interrupt, waits for its signalGracePeriod, and then calls
+			// Terminate.
```

Comment:
should we log something for this cancel, we have a few things happening at the same time here with go funcs ect, would be good to log so we kind of know what happend when. 

Staring at this code I don't know what will happen when so maybe some log messages would help.